app/controllers/api/v1/people_controller.rb

```diff
@@ -4,7 +4,14 @@ module Api
   module V1
     class PeopleController < ApiController
       def index
+        person_nomis_prison_number = filter_params[:nomis_offender_no]
```

Comment:
Is there a reason for this? I mean other than liking/disliking the naming. Maybe we can fix this at the source in the filter name? the variable is used only once apparently in the next line.

Reply:
it is used twice, the assignment make things more readable imo